autotx/utils/agent/define_tasks.py

```diff
@@ -6,23 +6,111 @@
+                    "agent": "send-tokens",
+                    "expected_output": "Current USDC balance of 0x5d15311D760511d89cFad67404131cdc155E9FDB",
+                    "context": null,
+                    "extra_information": "Use Get ERC20 balance tool with USDC token symbol to check current USDC balance of user's address."
+                }},
+                {{
+                    "task": "Calculate ETH amount needed to swap for 100 USDC",
+                    "agent": "swap-tokens",     
+                    "expected_output": "The amount of ETH needed to execute a swap to receive 100 USDC",
+                    "context": null,
+                    "extra_information": "Assuming the swap service or agent has the capability to estimate, based on existing market rates, how much ETH is required."
+                }},
+                {{
+                    "task": "Build transactions needed to execute the swap from ETH to 100 USDC",
+                    "agent": "swap-tokens",
+                    "expected_output": "Transactions prepared for swapping ETH to exactly 100 USDC",
+                    "context": [2],
+                    "extra_information": "Use the calculated ETH amount from the previous step to prepare the necessary transactions for the swap, ensuring that the exact_input is set to True since we want to receive an exact amount of USDC."
+                }}
+            ]
+        }}
+        
+        Goal: First, swap Ethereum (ETH) to 0.05 Wrapped Bitcoin (WBTC), ensuring the right amount of ETH is swapped for exactly 0.05 WBTC. After successfully swapping to WBTC, proceed to swap 0.05 WBTC to 1000 USD Coin (USDC), making sure to receive exactly 1000 USDC from the swap. Following these swaps, transfer 50 USDC to the Ethereum Name Service (ENS) domain 'vitalik.eth'.
+        Output:
+        {{
+            "tasks": [
+                {{
+                    "task": "Swap the required amount of Ethereum (ETH) for exactly 0.05 Wrapped Bitcoin (WBTC)",
+                    "agent": "swap-tokens",
+                    "expected_output": "A successful swap transaction from ETH to WBTC where the user ends up with exactly 0.05 WBTC",
+                    "context": null,
+                    "extra_information": "Use the exact_output mode for the swap, ensuring the exact amount of 0.05 WBTC is received. Calculate the required ETH amount based on current market rates."
+                }},
+                {{
+                    "task": "Swap exactly 0.05 Wrapped Bitcoin (WBTC) for 1000 USD Coin (USDC)",
+                    "agent": "swap-tokens",
+                    "expected_output": "A successful swap transaction from WBTC to USDC where the user ends up with exactly 1000 USDC",
+                    "context": null,
+                    "extra_information": "Use the exact_output mode for the swap, ensuring the exact amount of 1000 USDC is received. The exact amount of 0.05 WBTC is to be swapped."
```

Comment:
Same as previous comment, maybe we should tweak this to say that it should buy 1000 USDC with WBTC (Rather than saying the amount of WBTC must be 0.05)